apps/prairielearn/src/tests/groupRole.test.sql

```diff
@@ -38,14 +38,18 @@ WHERE
 
 -- BLOCK select_group_user_roles
 SELECT
-  gur.user_id,
+  gu.user_id,
   gur.group_role_id
 FROM
-  group_user_roles AS gur
-  JOIN groups AS gr ON gur.group_id = gr.id
-  JOIN group_configs AS gc ON gc.id = gr.group_config_id
+  group_configs AS gc
+  JOIN groups AS g ON g.group_config_id = gc.id
+  JOIN group_users AS gu ON gu.group_id = g.id
+  JOIN group_user_roles AS gur ON gur.group_user_id = gu.id
```

Comment:
This query was already updated to use group_user_id, since it's in tests and not expected to run in production.